snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ExportRequestBuilder.java

```diff
@@ -41,9 +42,15 @@
 	private boolean extensionOnly;
 	private String namespace;
 	private Collection<String> refSets = Collections.emptySet();
+	private Collection<String> componentTypes = Collections.emptySet();
```

Comment:
Does the default empty set mean that everything will be exported?

Reply:
I guess it doesn't matter right now 😃

Reply:
Yes, right now we use it only to detect single refset export. It can be used later to filter the content by component type, unless we rewrite the whole export. 😄